utils.py

```diff
@@ -498,6 +538,14 @@ def require_valid_name(name, name_type, allow_empty=False):
         name_type: str. A human-readable string, like 'the exploration title' or
             'a state name'. This will be shown in error messages.
         allow_empty: bool. If True, empty strings are allowed.
+
+    Raises:
+        Exception: Raised when name isn't a string.
```

Comment:
Again, no need to have 'Raises' in all these definitions, since the heading is 'Raises'. Just have the exception message itself here and elsewhere.